specs/tile_bag_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'minitest/autorun'
+require 'minitest/reporters'
+require 'minitest/skip_dsl'
+
+require_relative '../lib/tile_bag'
+require 'pry'
+
+Minitest::Reporters.use! Minitest::Reporters::SpecReporter.new
+
+describe 'TileBag' do
+  before do
+    @tile_bag = Scrabble::TileBag.new
+  end
+
+  describe 'initialize' do
+    it "Set-up the instance with a collection of all default tiles" do
+      @tile_bag.must_respond_to :tiles
+      @tile_bag.tiles.must_be_instance_of Array
+    end
+  end
+
+  describe 'draw_tiles' do
```

Comment:
What happens when you draw all the tiles?

Also you need to check that the number of each drawn tile has gone down.